Client/src/Components/Inputs/Field/index.jsx

```diff
@@ -56,7 +57,7 @@ const Field = forwardRef(
 		return (
 			<Stack
 				gap={theme.spacing(2)}
-				className={`field field-${type}`}
+				className={`${className ?? `field field-${type}`}`}
```

Comment:
we need the extra className not only to override, but also to hide the error , since we have changed the design to display single error msg for all the inputs errors